release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -177,6 +177,16 @@ For more information, see link:https://kubernetes.io/docs/concepts/services-netw
 
 [id="ocp-4-11-osdk"]
 === Operator development
+==== Performance Addon Operator functions moved to the Node Tuning Operator
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator, and all related artifacts, on startup. 
+
+[NOTE]
+====
+You must still use the `performance-addon-operator-must-gather` image when running the `must-gather` command with the Performance Profile Creator.
+====
+
+For more information, see xref:../operators/operator-reference.adoc#about-node-tuning-operator_platform-operators-ref[Node Tuning Operator].
```

Comment:
You could consider moving this line to be before the note, because the cross-reference seems to relate more to the main paragraph in this section, rather than the contents of the note.